qiskit/assembler/assemble_schedules.py

```diff
@@ -17,25 +17,23 @@
 from typing import Any, Dict, List, Tuple
```

Comment:
Are there any nontrivial changes in this file?

Reply:
Nope, other than maybe some deprecated imports being switched to the new ones. Same with the plotter changes.